docs/docs/eslint.md

```diff
@@ -1,33 +1,33 @@
 ---
-title: Using ESLint
+título: Usando ESLint
```

Comment:
```suggestion
title: Usando ESLint
```

Reply:
Thank you for the suggested changes. I just finished changing it. Please let me know what to do next.